docs/QUICKSTART.md

```diff
@@ -38,21 +61,38 @@ tufrepo/
 
     3 directories, 11 files
 ```
-The new file `testfile` is added and metadata is updated in the `tufrepo` directory.
+
+The new file `testfile` is added to the repository, and metadata is updated in
+the `tufrepo` directory.  The Targets metadata (`targets.json`) now includes
+the file size and hashes of the `testfile` target file, and this metadata is
+signed by the Targets role's key, so that clients can verify that metadata
+about `testfile` and then verify `testfile` itself.
+
 
 **Step (3)** - Serve the repo
+
+We'll host a toy http server containing the `testfile` update and the
+repository's metadata.
+
 ```Bash
 $ cd "tufrepo/"
+$ python3 -m http.server 8001
+
+# or, if you are using Python2:
 $ python -m SimpleHTTPServer 8001
 
-or with Python 3...
-$ python3 -m http.server 8001
 ```
 
-**Step (4)** - Fetch a target file from the repo.  The client downloads
-any required metadata and the requested target file.
+**Step (4)** - Obtain and verify the `testfile` update on a client.
```

Comment:
minuscule nit-pick: You end some heading sentences with a period and other not.

Reply:
Fixed